WORKSPACE

```diff
@@ -1,18 +1,23 @@
+workspace(name = "angular_devkit_src")
+
 git_repository(
     name = "build_bazel_rules_nodejs",
     remote = "https://github.com/bazelbuild/rules_nodejs.git",
     tag = "0.3.1",
 )
 
-load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories")
+load("@build_bazel_rules_nodejs//:defs.bzl", "check_bazel_version", "node_repositories")
+
+check_bazel_version("0.8.1")
```

Comment:
bazel 0.9 is out now, and resolves the issue that caused us some trouble with rxjs getting included in `bazel build ...` so maybe we should require 0.9.0?

Reply:
ngcontainer appears to only have 0.8.1